frontend-react/src/utils/CustomRenderUtils.tsx

```diff
@@ -128,6 +133,16 @@ export const renderWithQueryProvider = (
         ...options,
     });
 
+// for testing components that need access to feature flags
+export const renderWithFeatureFlags = (
```

Comment:
Should we add this to the "FullAppWrapper" custom renderer? I'm back and forth on it. On one hand, we'd have to mock it whenever we use that full app wrapper, but on the other, it's more in line with how the app will run.

Reply:
i don't see any reason not to include it there. If the test isn't using the feature flags hook then nothing should need to be updated, and all our tests still pass if I add it in